Vagrantfile

```diff
@@ -70,8 +70,15 @@ Vagrant.configure("2") do |config|
 					deprecated_extensions << "Please change #{extension} to #{new_extension} in your yaml configuration file.\n"
 				end
 			end
+			trigger.warn = "#{deprecated_extensions}"
+		end
```

Comment:
This isn't directly related to the stated issue, but moving this warning up a block prevents it from showing an empty

![image](https://user-images.githubusercontent.com/442115/85172796-c968e500-b23f-11ea-8d27-bd939cdaece8.png)

if there are no deprecated extensions to show.